src/Tizen.NUI/src/public/Window.cs

```diff
@@ -1310,5 +1310,52 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
         {
             Interop.Window.delete_Window(swigCPtr);
         }
+
+        /// <summary>
+        /// FrameCallbackType
```

Comment:
Type of callback which is called when the frame rendering is done by the graphics driver.